python/package/CMakeLists.txt

```diff
@@ -6,27 +9,35 @@ set(OPTIMA_PYTHON_PACKAGE_PATH ${CMAKE_CURRENT_BINARY_DIR}/optima)
 
 # Configure the setup.py file
 set(SETUP_PY_IN ${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in)
-set(SETUP_PY ${OPTIMA_PYTHON_PACKAGE_PATH}/setup.py) # ensure setup.py is inside the package dir
+set(SETUP_PY ${CMAKE_CURRENT_BINARY_DIR}/setup.py) # ensure setup.py is inside the package dir
 configure_file(${SETUP_PY_IN} ${SETUP_PY})
 
 # Create a custom target to build the python package during build stage
+#add_custom_target(optima-setuptools ALL
+#    COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}/optima ${OPTIMA_PYTHON_PACKAGE_PATH}
+#    COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:optima4py> ${OPTIMA_PYTHON_PACKAGE_PATH}
+#    COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} --quiet install --force --prefix=${CMAKE_BINARY_DIR}
+#    WORKING_DIRECTORY ${OPTIMA_PYTHON_PACKAGE_PATH})
```

Comment:
I have commented here to try the same code as for Reaktoro. Actually, the problem is that the `optima` installation should be in `${CONDA_PREFIX}\Lib\site-packages`, but it has been installed in `${CONDA_PREFIX}\Library\lib\site-packages` or elsewhere.